### PR TITLE
Make link in sublog for comment deletion point to the comment

### PR DESCRIPTION
**Pull request by happy-river**
Currently when an admin or mod deletes or undeletes a comment, the sub log entry gets a link pointing to the post.  This is not helpful for a mod who wants to see which of possibly many comments was deleted or undeleted.  Instead of a link to the post, put the permalink for the affected comment in the new sub log entry.

### CODE REVIEW

File: app/views/do.py [hunk 1]
```diff
@@ -2661,24 +2661,35 @@ def delete_comment():
             comment = SubPostComment.get(SubPostComment.cid == form.cid.data)
         except SubPostComment.DoesNotExist:
             return jsonify(status="error", error=_("Comment does not exist"))
-        post = SubPost.get(SubPost.pid == comment.pid)
+        sub = (
+            SubPost.select(SubPost.sid, Sub.name)
+            .join(Sub)
+            .where(SubPost.pid == comment.pid)
+            .dicts()
+        )[0]
```
Comment (Polsaker) [inline app/views/do.py:2669]: It's probably more elegant to do a 

```suggestion
        sub = SubPost.select(SubPost.sid, Sub.name)
            .join(Sub)
            .where(SubPost.pid == comment.pid)
            .get()
```

File: app/views/do.py [hunk 2]
```diff
@@ -2715,19 +2726,30 @@ def undelete_comment():
         except SubPostComment.DoesNotExist:
             return jsonify(status="error", error=_("Comment does not exist"))
 
-        post = SubPost.get(SubPost.pid == comment.pid)
+        sub = (
+            SubPost.select(SubPost.sid, Sub.name)
+            .join(Sub)
+            .where(SubPost.pid == comment.pid)
+            .dicts()
+        )[0]
```
Comment (Polsaker) [inline app/views/do.py:2734]: Same as above

File: app/views/do.py (hunk 1 above)
Comment (happy-river) [reply app/views/do.py]: That version doesn't work, but swapping the models between the `select` and the `join` does.

File: app/views/do.py (hunk 2 above)
Comment (happy-river) [reply app/views/do.py]: Same as above, pushed a revised version.

**Pull request merged**